service/controller/app/v1/key/key.go

```diff
@@ -65,6 +65,18 @@ func InCluster(customResource v1alpha1.App) bool {
 	return customResource.Spec.KubeConfig.InCluster
 }
 
+func KubeConfigFinalizer(customResource v1alpha1.App) string {
+	return fmt.Sprintf("%s/%s", label.FinalizerPrefix, customResource.GetName())
```

Comment:
Technically the finalizer prefix is not a label. I would not add it to the label package, but just add it here, maybe even inline.

Reply:
Sorry I don't get what you mean in here, how we can put it as inline?

Reply:
I thought about something like this. And then drop the changes in the label package. 

```suggestion
	return fmt.Sprintf("app-operator.giantswarm.io/%s", customResource.GetName())
```